samples/BitbucketStronglyTypedCoreReceiver/BitbucketStronglyTypedCoreReceiver.csproj

```diff
@@ -1,10 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>$(StandardTestWebsiteTfms)</TargetFrameworks>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore" Version="$(MicrosoftAspNetCorePackageVersion)" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.DataAnnotations" Version="$(MicrosoftAspNetCoreMvcDataAnnotationsPackageVersion)" />
```

Comment:
Why is this required - desktop support?

Reply:
The main WebHooks projects only depend on Microsoft.AspNetCore.Mvc. So, yes, need more when running on full framework.

Reply:
Oops, Mvc**.Core**